src/components/Forms/LoginForm/LoginForm.tsx

```diff
@@ -24,6 +25,20 @@ export const LoginForm = ({ setPageType }) => {
   const handleSubmit = async (event) => {
```

Comment:
I'd also move handle submit function to the separate services file for ease of testing, moreover this fucntion definition will be re-declared on each component re-render (and we want to avoid that).